aws/smithy-go
# ABC

- Add standard lib functions (#416)
- add endpoint params codegen (#421)
- Add code-generation for EndpointProvider (EndpointResolver) (#424)
- Add endpoints2.0 tests codegen (#427)
- Add deprecated option to config field resolvers (#428)
- Add endpoints2.0 middleware and param binding generation. (#430)
- Add hooks for consumer written auth scheme resolution (#431)
- make endpointresolver middleware object names public for otehr middlwares to identify it
- raw; forcewrite me later
- Update URI encoding to support URL.RawPath consistently

*Issue #, if available:*

*Description of changes:*


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
